Malaria_analysis.ipynb

```diff
@@ -0,0 +1,2105 @@
+{
```

Comment:
<p>Looks good! I'm glad you're learning both matplotlib and seaborn, they can both be helpful packages for you to learn. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ihmeuw/vivarium_research/pull/1713/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>